### PR TITLE
[IMPROVEMENT] Add dialog confirmation for logout

### LINKED ISSUE #1715: [NEW] Logout Confirmation
## Description

The app should ask for confirmation before Logout.

## Devices and Versions

Your Rocket.Chat.Android version: 2.6.1
Mobile device model and OS version:
Xiaomi Redmi Note 4 - Android 7.0
## Steps to reproduce
Just press LOGOUT

### PR DESCRIPTION
**Pull request by shubhsherl**
@RocketChat/android

Closes #1715 
Add Logout Confirmation.

roc

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/main/presentation/MainPresenter.kt [hunk 1]
```diff
@@ -163,6 +164,13 @@ class MainPresenter @Inject constructor(
         }
     }
 
+    fun showLogoutDialog(context: Context) {
+        val builder = AlertDialog.Builder(context)
+        builder.setMessage("Are you sure ?")
```
Comment (rafaelks) [inline app/src/main/java/chat/rocket/android/main/presentation/MainPresenter.kt]: Please use localised strings here

**Review by rafaelks — CHANGES_REQUESTED** [review]
Thanks for the PR! Just need to be localized.

**Comment by shubhsherl** [comment]
@rafaelks  PR updated.

**Comment by rafaelks** [comment]
@shubhsherl Could you please change the text from `Are You Sure?` to `Are you sure you want to logout?`

**Review by divyanshub024 — COMMENTED** [review]
@shubhsherl It's prefered to add a title in a dialog. So title may be `Log out` and message may be `Are you sure you want to logout?`

**Pull request closed without merging**